src/tests/ftest/control/dmg_telemetry_net.py

```diff
@@ -0,0 +1,166 @@
+        # telemtry system, which may or may not be zero.
+        req_timeouts_start = 0
+
+        data = {}
+        try:
+            data = self.telemetry.get_metrics(metrics[0])
+        except (CommandFailure, DaosTestError) as excep:
+            self.log.info("self.telemetry.get_metrics failed on at least one "
+                          "host with '{}', but may have succeeded elsewhere."
+                          .format(repr(excep)))
+            pass
+
+        for host, value in data.items():
+            for metric in value[metrics[0]]["metrics"]:
+                req_timeouts_start += metric["value"]
+
+        req_timeouts_finish = req_timeouts_start
+
+        # set params
+        targets = self.params.get("targets", "/run/server_config/*")
+        rank = self.params.get("rank_to_kill", "/run/testparams/*")
+
+        # create pool
+        self.add_pool(connect=False)
+
+        # make sure pool looks good before we start
+        checks = {
+            "pi_nnodes": len(self.hostlist_servers),
+            "pi_ntargets": len(self.hostlist_servers) * targets,
+            "pi_ndisabled": 0,
+        }
+        self.assertTrue(
+            self.pool.check_pool_info(**checks),
+            "Invalid pool information detected before rebuild")
+
+        # create 1st container
+        self.add_container(self.pool)
+
+        # start mdtest run
+        thread = threading.Thread(target=self.execute_mdtest)
+        thread.start()
+        time.sleep(5)
+
+        self.server_managers[0].stop_ranks([rank[0]], self.d_log, force=True)
+        time.sleep(10)
+
+        # _req_timeout metric may not be on all hosts
+        self.telemetry.hosts.clear()
+        self.telemetry.hosts.add(self.hostlist_servers[0])
+
+        data = {}
+        try:
+            data = self.telemetry.get_metrics(metrics[0])
+        except (CommandFailure, DaosTestError) as excep:
+            self.log.info("self.telemetry.get_metrics failed on at least one "
+                          "host with {}, but may have succeeded elsewhere."
+                          .format(repr(excep)))
+            pass
+
+        for host, value in data.items():
+            for metric in value[metrics[0]]["metrics"]:
+                req_timeouts_finish += metric["value"]
+
+        req_timeouts_delta = req_timeouts_finish - req_timeouts_start
+        if req_timeouts_delta > 0:
+            self.log.info("Expected {} values to increase by more than 0 "
+                          "during this test, and they did: {}."
+                          .format(str(metrics[0]), str(req_timeouts_delta)))
+        else:
+            self.fail("Expected {} to increase by more than 0 during this "
+                      "test.".format(str(metrics[0])))
+
+        # wait a bit for mdtest to complete
+        try:
+            thread.join(timeout=5)
+        except Exception as excep:
+            self.log.info("MDtest threw an exception ({}), but this is not "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions